db/migrate/20210417041259_create_category_summaries.rb

```diff
@@ -0,0 +1,22 @@
+class CreateCategorySummaries < ActiveRecord::Migration[4.2]
+  def up
+    execute %[
+      create view category_summaries as
+        select
+            to_char(date, 'YYYY-MM') as month
+          , accounts.currency as currency
+          , category as name
+          , sum(value)
+          , expected_movement
+        from movements
+        inner join accounts on chargeable_id = accounts.id and chargeable_type = 'Account'
+        where kind = 'Outgo'
+        group by month, movements.category, currency, expected_movement
+        order by month, name;
```

Comment:
Just in case :man_dancing: 

```sql
select
    to_char(date, 'YYYY-MM') as month
  , accounts.currency as currency
  , category as name
  , (case
      when expected_movement = true then sum(movements.value)
      else 0.0
    end) as expected
  , (case
      when expected_movement = false then sum(movements.value)
      else 0.0
    end) as unexpected
from movements
inner join accounts on chargeable_id = accounts.id and chargeable_type = 'Account'
where kind = 'Outgo'
group by month, movements.category, currency, expected_movement
order by month, name;
```